infrastructure/cdn-in-a-box/traffic_ops/Dockerfile

```diff
@@ -20,129 +20,76 @@
 # Based on CentOS 8
 ############################################################
 
-# Keep the trafficops-common-deps in Dockerfile the same as
-# trafficops-common-deps in Dockerfile-go to cache the same
-# layer.
 ARG RHEL_VERSION=8
-FROM centos:${RHEL_VERSION} as trafficops-common-deps
+FROM centos:${RHEL_VERSION}
 ARG RHEL_VERSION=8
 # Makes RHEL_VERSION available in later layers without needing to specify it again
 ENV RHEL_VERSION=$RHEL_VERSION
 
 RUN if [[ "${RHEL_VERSION%%.*}" -eq 7 ]]; then \
-        yum -y install dnf || exit 1; \
-    fi
+		yum -y install dnf || exit 1; \
+	fi
 
 RUN set -o nounset -o errexit && \
-    mkdir -p /etc/cron.d; \
-    if [[ "${RHEL_VERSION%%.*}" -eq 7 ]]; then \
-        use_repo=''; \
-    else \
-        use_repo='--repo=pgdg96'; \
-    fi; \
-    dnf -y install "https://download.postgresql.org/pub/repos/yum/reporpms/EL-${RHEL_VERSION%%.*}-x86_64/pgdg-redhat-repo-latest.noarch.rpm"; \
-    dnf -y $use_repo -- install postgresql96; \
-    dnf -y install epel-release; \
-    dnf -y install      \
-        jq              \
-        bind-utils      \
-        net-tools       \
-        gettext         \
-        perl-JSON-PP    \
-        mkisofs         \
-        isomd5sum       \
-        nmap-ncat       \
-        openssl         \
-        # Used to copy certs in "Shared SSL certificate generation" step
-        rsync;          \
-    dnf clean all
-
-FROM trafficops-common-deps as trafficops-perl-deps
+	mkdir -p /etc/cron.d; \
+	if [[ "${RHEL_VERSION%%.*}" -eq 7 ]]; then \
+		use_repo=''; \
+	else \
+		use_repo='--repo=pgdg96'; \
+	fi; \
+	dnf -y install "https://download.postgresql.org/pub/repos/yum/reporpms/EL-${RHEL_VERSION%%.*}-x86_64/pgdg-redhat-repo-latest.noarch.rpm"; \
+	dnf -y $use_repo -- install postgresql96; \
+	dnf -y install epel-release; \
+	dnf -y install      \
+		jq              \
+		bind-utils      \
+		net-tools       \
+		gettext         \
+		perl-JSON-PP    \
+		perl-Crypt-ScryptKDF  \
```

Comment:
Do we still need these perl packages?

OIC, I think this is required because of the components like `adduser.pl` since it's requiring the _Crypt::ScriptKDF_ library.

Should we look into removing the perl scripts as a separate issue in the future?

Reply:
> Should we look into removing the perl scripts as a separate issue in the future?

For sure. All of the Perl cryptography functions it uses are reimplemented in Go and used by traffic_ops_golang, so we should have everything we need to rewrite it.

Reply:
There are also a bunch of other scripts - like the checks - that have their own dependencies which were once part of the carton dependencies of the Perl version of Traffic Ops that we'll probably want to re-write (or remove, as appropriate) before too long.